src/ariel-os-native/Cargo.toml

```
@@ -0,0 +1,47 @@
+[package]
+name = "ariel-os-native"
+version = "0.1.0"
```

Seems like it would be easier if we kept it in sync with the other crates:

```suggestion
version = "0.2.0"
```